.idea/runConfigurations/entrypoint.xml

```diff
@@ -6,7 +6,7 @@
     <envs>
       <env name="PYTHONUNBUFFERED" value="1" />
     </envs>
-    <option name="SDK_HOME" value="$PROJECT_DIR$/venv/bin/python" />
+    <option name="SDK_HOME" value="/usr/local/bin/python3.9" />
```

Comment:
python could potentially be located within a different folder, it depends on the machine, we should not hardcode this. venv/bin/python will use the python that is installed with Xcode, at least this path is constant on MacOS